students/jemielity_kamil/lesson_08_regular_expressions/email_validator.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+
+def email_validator(string):
+    regex = re.compile(r'[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+.[a-zA-Z0-9]{2,3}')
+    mo = regex.search(string)
+    if mo is not None:
+        print(mo.group())
+    else:
+        print('Wrong email address')
+
+
+email_validator('aaa@aaa.pl')
```

Comment:
It's not a requirement here but please try to use pytest for verification
class TestEmailAddressValidator:
    @pytest.mark.parametrize("address", [
        'aaa@aaa.pl',
        'a-a-a@aaa.pl',
		...,
    ])
    def test_valid_email_address(self, address):
        assert email_validator(address)